aws/components/gateway/state.ftl

```diff
@@ -74,12 +75,15 @@
             [#break]
 
         [#case "router" ]
-            [#local resources += {
-                "transitGatewayAttachement" : {
-                        "Id" : formatResourceId(
+
+            [#local transitGatewayAttachementId = formatResourceId(
                             AWS_TRANSITGATEWAY_ATTACHMENT_RESOURCE_TYPE,
                             core.Id
-                        ),
+                        )]
+
+            [#local resources += {
+                "transitGatewayAttachement" : {
+                        "Id" : transitGatewayAttachementId,
```

Comment:
Same here -maybe go through and global update?

Reply:
fixeded